repositories/dbmodels/db_decisions_to_create.go

```diff
@@ -20,13 +20,13 @@ const TABLE_DECISIONS_TO_CREATE = "decisions_to_create"
 
 var DecisionToCreateFields = utils.ColumnList[DecisionToCreate]()
 
-func AdaptDecisionToCrate(db DecisionToCreate) models.DecisionToCreate {
+func AdaptDecisionToCrate(db DecisionToCreate) (models.DecisionToCreate, error) {
```

Comment:
Typo
```suggestion
func AdaptDecisionToCreate(db DecisionToCreate) (models.DecisionToCreate, error) {
```

And I'm not a fan of adding a "nil" error. It feels like the function is failable from the signature, but it do not happen.